### PR TITLE
Add molmass

### PR DESCRIPTION
**Pull request by n-elie**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by n-elie** [comment]
@conda-forge/help-python Ready for review

File: recipes/molmass/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        
+  - name: {{ name | lower }}-elements-gui
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - {{ pin_subpackage('molmass', exact=True) }}
+        - python
```
Comment (beenje) [inline recipes/molmass/meta.yaml:60]: No need to redefine python as it's part of `molmass` run requirement

**Review by beenje — CHANGES_REQUESTED** [review]

File: recipes/molmass/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        
+  - name: {{ name|lower }}-web
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - {{ pin_subpackage('molmass', exact=True) }}
+        - python
```
[the remark this answers is not in the document]
Comment (n-elie) [reply recipes/molmass/meta.yaml]: @beenje Thanks for review, I applied all the suggested changes. However, removing `python` from outputs requirements leads to the following error:
```
Error: Failed to render jinja template in /home/conda/staged-recipes-copy/recipes/molmass/meta.yaml:
'PYTHON' is undefined
```
Any idea on how could I fix this?

[the remark this answers is not in the document]
Comment (n-elie) [reply recipes/molmass/meta.yaml]: Ok I found that I need to remove python from `run` section but not from `host`

**Pull request merged**